src/compiler/sys/environment.ts

```diff
@@ -1,16 +1,13 @@
-export const IS_DENO_ENV = typeof Deno !== 'undefined';
-
 export const IS_NODE_ENV =
-  !IS_DENO_ENV &&
   typeof global !== 'undefined' &&
   typeof require === 'function' &&
   !!global.process &&
   typeof __filename === 'string' &&
   (!(global as any as Window).origin || typeof (global as any as Window).origin !== 'string');
 
-export const OS_PLATFORM = IS_NODE_ENV ? process.platform : IS_DENO_ENV ? Deno.build.os : '';
```

Comment:
`Deno.build.os` could return the string `"windows"` which is why we can safely remove the check for it in the line below (in fact, the compiler yells at us because `"windows"` is not a valid `process.plaform` value